client/src/test/java/se/fortnox/reactivewizard/client/HttpClientTest.java

```diff
@@ -849,6 +885,8 @@ public void shouldCloseConnectionOnTimeoutDuringContentReceive() throws URISynta
     @Test
     public void shouldShouldNotGivePoolExhaustedIfServerDoesNotCloseConnection() throws URISyntaxException {
```

Comment:
Was this somehow related to your changes, or did it resolve some previous issue?

Reply:
This resolved an issue where this test turned off logging for my test case which made it fail.